bedrock/security/templates/security/bug-bounty/web-hall-of-fame.html

```diff
@@ -28,7 +29,7 @@ <h1 itemprop="name" class="title-shadow-box">Web And Services Bug Bounty Hall of
 
       <p>On behalf of the Mozilla and the millions of people who visit our sites, use Firefox and our other products we would like to thank them for their hard work in helping to make us more secure.</p>
 
-      <p>As of this date, we have paid out almost <b>$1,000,000</b> across all of our bounties. Congratulations to everybody who has participated!</p>
+      <p>As of this date, we have paid out over <b>$1,600,000</b> across all of our bounties. Congratulations to everybody who has participated!</p>
```

Comment:
Not including a date always it to always be accurate and also reduces the number of updates we need to do.